templates/partial/_data-navigation.html

```diff
@@ -0,0 +1,49 @@
+<div id="g-navigation" class="p-navigation is-dark">
+  <div class="p-navigation__row global-nav-row">
+    <div class="p-navigation__banner">
+      <a class="p-navigation__item p-global-nav-anchors" href="/">Canonical</a>
+      <a href="#g-navigation" class="p-navigation__toggle--open p-global-nav-anchors" title="menu">Menu</a>
+      <a href="#g-navigation-closed" class="p-navigation__toggle--close p-global-nav-anchors" title="close menu">Close menu</a>
+    </div>
+    <nav class="p-navigation__nav">
+      <ul class="p-navigation__items"></ul>
+      <ul class="p-navigation__items global-nav"></ul>
+    </nav>
+  </div>
+</div>
+<header id="navigation" class="p-navigation is-dark" style="z-index: 9;">
+  <div class="row data-fabric-menu">
+    <div class="p-navigation__banner col-3">
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="/data">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="Canonical" width="95">
+          </div>
+          <span class="p-navigation__logo-title">Data Fabric</span>
+        </a>
+      </div>
+      <a href="#navigation" class="p-navigation__toggle--open" title="Toggle navigation"><i class="p-icon--chevron-down is-light"></i></a>
+      <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
```

Comment:
please start from the capital letter in `title` as it's visible to users on hover